static/app/views/replays/replays.tsx

```diff
@@ -66,7 +60,7 @@ function Replays(props: Props) {
       id: '',
       name: '',
       version: 2,
-      fields: ['eventID', 'project', 'timestamp', 'user.display', 'url'],
+      fields: ['eventID', 'project', 'user.display', 'url', 'timestamp', 'id'],
```

Comment:
is id and eventID different?

Reply:
Yes, it is a different id from eventID, in my first approach think it was the same, but then realize eventID is empty sometimes, and maybe useless?

Reply:
think also we can remove eventID to the current query. (Is not used in any place of the query or the table)

Reply:
Yeah let's follow-up in a new PR and remove if it's not used.